src/app/main.cpp

```diff
@@ -233,16 +236,24 @@ int main(int argc, char *argv[])
         app->setAttribute(Qt::AA_DontShowIconsInMenus);
 #endif
 
+        if (!firstTimeUser) {
+            handleChangedDefaults(DefaultPreferencesMode::Legacy);
```

Comment:
Unless I am missing something:
1. New user, this check is skipped for the first run
2. The other check writes explicit value (new default)
3. New user restarts qbt and enters this check
4. Setting value is reverted to old default now

Reply:
Yes that's correct and this is also changing the already stored value for old users. If old user had queue disabled, its enabling it.

Reply:
PR updated. Can you try it now?
I don't know why I missed the check for explicitly written settings last time...

Reply:
tested and working as expected.